src/rubintv/handlers/external/endpoints.py

```diff
@@ -413,23 +413,29 @@ async def events(request: web.Request) -> dict[str, Any]:
         bucket = request.config_dict["rubintv/gcs_bucket"]
         camera = cameras[request.match_info["camera"]]
         channel_name = request.match_info["channel"]
-        date = request.match_info["date"]
+        the_date = request.match_info["date"]
         seq = request.match_info["seq"]
         channel = camera.channels[channel_name]
         title = build_title(
-            camera.name, channel.name, date, seq, request=request
+            camera.name, channel.name, the_date, seq, request=request
         )
         prefix = channel.prefix
         prefix_dashes = prefix.replace("_", "-")
-        event = Event(
-            f"https://storage.googleapis.com/{bucket.name}/{prefix}/"
-            f"{prefix_dashes}_dayObs_{date}_seqNum_{seq}.png"
+        whole_prefix = (
+            f"{prefix}/{prefix_dashes}_dayObs_{the_date}_seqNum_{seq}"
         )
+        # empty URL string is handled in the template
+        url = ""
+        if blobs := list(bucket.list_blobs(prefix=whole_prefix)):
```

Comment:
Is there a bucket method for getting a single resource? It seems odd (though not broken) to be listing objects and getting the first one when in fact you know the full URL for the resource anyhow.

Reply:
Sorry about the busy nature of the commits! 
I don't understand why this line is still there though. It was already changed in the way you suggest in this commit:
https://github.com/lsst-sqre/rubin-tv/pull/105/commits/848227fd0db6197b83a217c3bb0bf5ff749c9f13